app/javascript/components/test-utils.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
```

Comment:
We should move this files under `app/javascript/spec/helpers` instead of putting test-only helpers in components.